src/assets/content/dictionary/flowchart.md

```diff
@@ -1,6 +1,9 @@
 ---
 title: Flowchart
-definition:
-sources: []
-perspectives: []
+definition: a diagram that shows step-by-step progression through a procedure or system especially using connecting lines and a set of conventional symbols
+sources:
+- sourceurl: https://www.merriam-webster.com/dictionary/flowchart
+perspectives:
+- meaning: which step/functionality work after which step. this roadmap is flowchart.
```

Comment:
```suggestion
- meaning: which step/functionality comes after which step. A roadmap is a flowchart
```
suggestion: I've made suggestion for clarity and syntax for the perspective. You can use the Github UI to accept the suggestion here, or make an extra commit to this PR.